addons/mail/static/src/scss/discuss.scss

```diff
@@ -327,9 +327,18 @@
         .o_preview_title {
             display: flex;
             .o_preview_name {
-                flex: 0 1 auto;
+                flex: 1 0 auto;
```

Comment:
`flex: 0 0 auto;` (the preview name should not grow, otherwise the counter is not next to it)